Pages/Page 2_Big Picture.md

```diff
@@ -0,0 +1,170 @@
+        2  * 96.120.49.33 (96.120.49.33)  30.033 ms  30.066 ms
+        3  te-0-3-0-1-sur02.webster.mn.minn.comcast.net (68.85.167.149)  30.553 ms  32.776 ms  32.785 ms
+        4  te-0-7-0-12-ar01.roseville.mn.minn.comcast.net (69.139.219.134)  32.778 ms  36.059 ms te-0-7-0-13-ar01.roseville.mn.minn.comcast.net (68.87.174.185)  35.686 ms
+        5  he-1-13-0-0-cr01.350ecermak.il.ibone.comcast.net (68.86.94.81)  42.354 ms *  43.213 ms
+        6  be-10206-cr01.newyork.ny.ibone.comcast.net (68.86.86.225)  63.574 ms  59.894 ms  62.441 ms
+        7  pos-2-5-0-0-cr01.dallas.tx.ibone.comcast.net (68.86.85.25)  64.643 ms  64.659 ms  64.653 ms
+        8  he-3-14-0-0-cr01.dallas.tx.ibone.comcast.net (68.86.85.1)  67.039 ms  67.079 ms  67.056 ms
+        9  he-0-10-0-0-pe07.ashburn.va.ibone.comcast.net (68.86.83.66)  63.013 ms  67.045 ms  88.273 ms
+        10  * * *
+        11  * * *
+        12  * * *
+        13  * * *
+        14  * * *
+        15  * * *
+        16  * * *
+        17  * * *
+        18  * * *
+        19  * * *
+        20  * * *
+        21  * * *
+        22  * * *
+        23  * * *
+        24  * * *
+        25  * * *
+        26  * * *
+        27  * * *
+        28  * * *
+        29  * * *
+        30  * * *
+
++ Line 1 is my computer server address.
++ Line 2 is my router server address.
++ Lines 3-30, the request is moving through Comcast's network of servers to get the information.
+
+When you enter [https://github.com](httpss://github.com) in the address bar, your browser makes a GET request to the GitHub server. There are several types of HTTP requests, but GET is the one that asks the server to send data from a specified resource.  The server sends back an HTML page and the Web browser turns the HTML into the web page you see through a process called rendering. You can see the HTML by right-clicking on the page and selecting `View Page Source`.
+
+
+### HTML Proper
+
+
+HTML stands for "HyperText Markup Language".
+
+Hypertext means documents can contain links to other pages or images. The structure of the HTML document is encoded using a markup language consisting of opening and closing elements (also called tags). Each segment of text is formatted according to the type of tag (`<body>, <title>, <h2>,` etc). Each segment of text needs an opening tag, `<body>`, and a closing tag, `</body>`. The `/` signifies the end of the formatting for that segment.
+
+#### Let's see it in action
+Open a text editing program and enter the following text: 
+
+`<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sample HTML Page</title>
+  </head>
+  <body>
+    This is a sample html page.  This is more text.
+  </body>
+</html>`
+
+Save the file as 'sample.html'. Then open the file in the web browser. On Windows or a Mac, double-click the file to open it in the default browser, or right-click and select `Open With`. In your browser you should see something like this,
+
+* <<<_insert screenshot_>>>
```

Comment:
Just the rendered HTML
![screen shot 2015-06-10 at 1 18 26 pm](https://cloud.githubusercontent.com/assets/5800836/8090385/48f45f68-0f73-11e5-8305-abaa1dbd8117.png)

Inspecting the element showing the source
![screen shot 2015-06-10 at 1 19 51 pm](https://cloud.githubusercontent.com/assets/5800836/8090408/66b16cbc-0f73-11e5-96d3-f2d871ca946e.png)

Viewing the source
![screen shot 2015-06-10 at 1 20 09 pm](https://cloud.githubusercontent.com/assets/5800836/8090421/81840946-0f73-11e5-8ccc-e646868a11a7.png)